tests/integration/test_cloudformation.py

```diff
@@ -990,6 +992,20 @@ def test_cfn_handle_sqs_resource(self):
         rs = sqs.get_queue_url(QueueName=queue_name)
         self.assertEqual(rs['ResponseMetadata']['HTTPStatusCode'], 200)
 
+        queue_url = rs['QueueUrl']
+
+        rs = sqs.get_queue_attributes(
+            QueueUrl=queue_url,
+            AttributeNames=['All']
+        )
+
+        attributes = rs['Attributes']
+
+        self.assertIn('ContentBasedDeduplication', attributes)
+        self.assertIn('FifoQueue', attributes)
+        self.assertEqual(attributes['ContentBasedDeduplication'], 'True')
```

Comment:
Nit: Looks like the value of this property should be `'true'`, instead of `'True'`:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.ServiceResource.create_queue

(same for `FifoQueue` below)